prog/dftb+/prg_dftb/main.F90

```diff
@@ -424,10 +418,10 @@ subroutine runDftbPlus()
           potential%intBlock = potential%intBlock + potential%extBlock
         end if
 
-        call getEnergies(sccCalc, qOutput, q0, chargePerShell, species, tEField, tXlbomd,&
-            & tDftbU, tDualSpinOrbit, rhoPrim, H0, orb, neighborList, nNeighbor, img2CentCell,&
-            & iSparseStart, cellVol, extPressure, TS, potential, energy, thirdOrd, qBlockOut,&
-            & qiBlockOut, nDftbUFunc, UJ, nUJ, iUJ, niUJ, xi)
+!        call getEnergies(sccCalc, qOutput, q0, chargePerShell, species, tEField, tXlbomd,&
+!            & tDftbU, tDualSpinOrbit, rhoPrim, H0, orb, neighborList, nNeighbor, img2CentCell,&
+!            & iSparseStart, cellVol, extPressure, TS, potential, energy, thirdOrd, qBlockOut,&
+!            & qiBlockOut, nDftbUFunc, UJ, nUJ, iUJ, niUJ, xi)
 
```

Comment:
That would unfortunately screw up the output completely, as the user would see only zeros for the energies during the scc-cycle. We would have probably to separate the energy calculation into an inner- and an outer-scc part.